tests/azure-test/basic/connect.go

```diff
@@ -0,0 +1,81 @@
+package basic
+
+import (
+	"fmt"
+	"time"
+
+	"k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/uuid"
+	clientset "k8s.io/client-go/kubernetes"
+	testutils "k8s.io/cloud-provider-azure/tests/azure-test/utils"
+	//imageutils "k8s.io/cloud-provider-azure/tests/azure-test/utils/image"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("test ", func() {
+	var cs clientset.Interface
+	basename := "test"
+	var ns *v1.Namespace
+	//configPath := "C:\\Users\\t-xinhli\\.kube\\config"
```

Comment:
remove these comments when it's ready for commit

Reply:
Removed